packages/neon-invoker/package.json

```diff
@@ -12,16 +12,16 @@
     "test": "ts-mocha src/**/*.spec.ts"
   },
   "dependencies": {
-    "@cityofzion/neon-core": "^5.2.0",
-    "@cityofzion/neon-js": "^5.2.0",
-    "@cityofzion/neo3-invoker": "^1.1.0"
+    "@cityofzion/neo3-invoker": "^1.1.0",
+    "@cityofzion/neon-core": "^5.3.0",
+    "@cityofzion/neon-js": "^5.3.0"
```

Comment:
I believe that maybe we could only use neon-core or neon-js, can you fix this pls? 😅